.circleci/config.yml

```diff
@@ -526,7 +546,8 @@ jobs:
       - restore_cache:
           keys:
             - cargocache-v2-contract_cyberpunk-rust:1.73-{{ checksum "Cargo.lock" }}
-      - check_contract
+      - check_contract:
+          min_version: "1.4"
```

Comment:
Now with this setup we can even mix a bit more and use 1.3 for this case

Reply:
Yep. And it seems to pass just fine with 1.3